packages/yourloops/test/integration/pages/patient-dashboard/patient-dashboard.hcp.spec.tsx

```diff
@@ -122,8 +128,8 @@ describe('Patient dashboard for HCP', () => {
   })
 
   it('should be able to manage medical reports', async () => {
+    mockDataAPI(emptyWeightData)
```

Comment:
Why did you change the mocked value for this test? Nothing you did should impact the medical reports, so you should not have to bring any modification to this test